.github/workflows/ci.yml

```diff
@@ -0,0 +1,82 @@
+---
+name: CI
+on:
+  push:
+    tags: ['v*']
+    branches: [master]
+  pull_request:
+    branches: ['**']
+jobs:
+  # Tests are currently failing because they rely on a live YouTube account
+  # Until that issue is resolved, the tests are not very useful. Disabling them
+  # in CI for now.
+  # test:
+  #   runs-on: ubuntu-latest
+  #   strategy:
+  #     fail-fast: false
+  #     matrix:
+  #       ruby:
+  #         - '2.1'
+  #         - '2.3'
+  #         - '2.4'
+  #         - '2.5'
+  #         - '2.6'
+  #         - '2.7'
+  #         - '3.0'
+  #         - '3.1'
+  #         - '3.2'
+  #         - '3.3'
+  #         - jruby-head
+  #         - truffleruby-head
+  #
+  #   steps:
+  #     - uses: actions/checkout@v4
+  #     - uses: ruby/setup-ruby@v1
+  #       with:
+  #         ruby-version: ${{ matrix.ruby }}
+  #         bundler-cache: true
+  #     - run: bundle exec rspec --format doc
+  #     - uses: codecov/codecov-action@v3
+  #       if: matrix.ruby == '3.2'
+  #       with:
+  #         files: coverage/coverage.xml
+
+  yard:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+          bundler-cache: true
+      - run: bundle exec yardoc --fail-on-warning
+
+  check_version:
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
```

Comment:
Does this mean the check happens for ruby of version `2.7`? That version EOL was like over a year ago

Reply:
This gem supports down to 2.1 at the moment according to the gemspec. Is using 2.7 to run the yard check a problem?

Reply:
I see.